clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/arrays/TransferArraysV1Metadata.scala

```diff
@@ -0,0 +1,151 @@
+package org.broadinstitute.clio.transfer.model.arrays
+
+import java.net.URI
+import java.time.OffsetDateTime
+
+import org.broadinstitute.clio.transfer.model.TransferMetadata
+import org.broadinstitute.clio.util.model.DocumentStatus
+
+case class TransferArraysV1Metadata(
+  autocallCallRate: Option[Double] = None,
+  autocallDate: Option[OffsetDateTime] = None,
```

Comment:
why OffsetDateTime?

Reply:
Why not OffsetDateTime?

Reply:
`OffsetDateTime` is what we currently use to store dates in elasticsearch.